Backend/index.js

```diff
@@ -0,0 +1,31 @@
+import "dotenv/config";
+import express from "express";
+import mongoose from "mongoose";
+import cors from "cors";
+
+import resumeRoutes from "./routes/api/v1/resume/index.js";
+
+const app = express();
+
+const PORT = process.env.PORT || 3030;
+
+app.use(cors({ origin: process.env.CORS_ORIGIN }));
+
+app.use(express.json());
+
+app.get("/", (req, res) => {
```

Comment:
Remove this path, as it's an internal API, we do not need to return any irrelevant content.

Reply:
Removed in commit d4dea4f